models/models.go

```diff
@@ -68,12 +85,12 @@ const EmptyEvent = 501
 
 // Cross detail for each transaction. Corresponds to CrossEvent in the contract.
 type CrossInfo struct {
-	Id             int        `json:"id" gorm:"primary_key,autoIncrement"`
+	Id             int        `json:"id" gorm:"primary_key,autoIncrement,index:idx_target_chain,priority:2"`
```

Comment:
any necessity for union index on (targetChain, id)?

Reply:
Claiming worker will query records by chain id with id ordered.